### DISCUSSION TITLE
How to know if a slickgrid has the focus

### DISCUSSION THREAD

**Discussion by carboncs16** [ghiscoding/Angular-Slickgrid#1004] [Q&A] [answered]
I want to know if a slickgrid has the focus. I have 3 slickgrid components and I want to listen keydown event for the kurrently focused slickgrid.

**Comment by ghiscoding** [comment, marked as answer]
I don't know the answer to that question, but my wild guess would be to use plain JavaScript and add an event listener to each grid. Please ask these type of questions on Stack Overflow, and doing a quick search reveals nice answers like this one [How do I find out which DOM element has the focus?](https://stackoverflow.com/questions/497094/how-do-i-find-out-which-dom-element-has-the-focus)
